### PR TITLE
Remove Correlation Vector

### PR DESCRIPTION
**Pull request by alfredoihernandez**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Remove Use of Correlation Vector in favor of OpenTelemetry Standards

## Does this introduce a breaking change

- [x] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- References https://github.com/retaildevcrews/wcnp/issues/804

### CODE REVIEW

**Review by gortegaMS — APPROVED** [review]
LGTM

**Comment by kforeverisback** [comment]
There are some whitespace issues (space vs tab).
We want to be consistent with rest of the repo.
Other than that, LGTM.

**Review by kforeverisback — APPROVED** [review]
Approving it, correlation vector removed and tested TraceID.

I will create a triage item to make sure all our line endings and whitespaces are consistent throughout the repo.

FYI:
To verify the proper line ending in git try the below:
```bash
# Set Whitespace error highlight in current project
# Use --global to set it globally
git config --local diff.wsErrorHighlight all

# The step below is optional
git config --local core.pager less

# On terminal env, the command below will show `^M` for CRLF
git diff main HEAD
# 
```

If using VSCode, try `Render Line Endings` extension to check the Line Endings.

File: src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs [hunk 1]
```diff
@@ -3,20 +3,18 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Net;
+using System.Diagnostics;
```
[the remark this answers is not in the document]
Comment (alfredoihernandez) [reply src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs:6]: fixing with following item: https://github.com/orgs/retaildevcrews/projects/12#card-83665653

File: src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs [hunk 2]
```diff
@@ -75,29 +73,29 @@ public RequestLogger(RequestDelegate next, IOptions<RequestLoggerOptions> option
                 });
         }
 
-        /// <summary>
-        /// Gets or sets CosmosDB name.
-        /// </summary>
+        /// <summary>
```
[the remark this answers is not in the document]
Comment (alfredoihernandez) [reply src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs:76]: fixing with following item: https://github.com/orgs/retaildevcrews/projects/12#card-83665653

**Pull request merged**